### PR TITLE
properly store binaries with kviator

### PR DESCRIPTION
**Pull request by fmeppo**
kviator can't store binary data (see ceph/ceph-docker #220 ).  This works around that limit via sharutils and uuencoding/decoding the monmap.

### CODE REVIEW

File: daemon/Dockerfile [hunk 1]
```diff
@@ -10,7 +10,7 @@ MAINTAINER Sébastien Han "seb@redhat.com"
 # patch is only needed until PR #7351 is merged in ceph/ceph (and backported
 # to the version used in ceph/ceph-docker)
 RUN apt-get update && apt-get -y install runit && \
-    apt-get -y install patch && \
+    apt-get -y install patch sharutils && \
```
Comment (leseb) [inline daemon/Dockerfile]: can you integrate the package in the previous so we don't call`apt-get` twice? (unless you have a good reason).

**Comment by leseb** [comment]
Just a minor comment other than that LGTM.
Thanks @fmeppo!

File: daemon/Dockerfile (hunk 1 above)
Comment (leseb) [inline daemon/Dockerfile]: oh wait, why isn't this part of base instead?

**Comment by fmeppo** [comment]
Cleaned up, should be good.

**Comment by leseb** [comment]
I know it's related to your patch and we can fix that later but I still wonder why we have installation step into the entrypoint.
We can fix that another PR though.

Thanks!

**Pull request merged**